D2L.Services.Core.WebApi/Service.cs

```diff
@@ -42,6 +43,30 @@ Type dependencyLoaderType
 				)
 			);
 		}
+		
+		public Service(

```

Comment:
What I was hoping for was two independent constructors:
 - one works off of a type (as it does today)
 - the other works off of an IDependencyResolver (your test passes in an instance of TestDependencyResolver)